instrumentation/opentelemetry-instrumentation-flask/src/opentelemetry/instrumentation/flask/__init__.py

```diff
@@ -388,13 +394,25 @@ def _start_response(status, response_headers, *args, **kwargs):
             duration_attrs_old = otel_wsgi._parse_duration_attrs(
                 attributes, _HTTPStabilityMode.DEFAULT
             )
+
+            if wrapped_app_environ.get(_ENVIRON_REQUEST_ROUTE_KEY, None):
+                duration_attrs_old[SpanAttributes.HTTP_ROUTE] = (
+                    wrapped_app_environ.get(_ENVIRON_REQUEST_ROUTE_KEY)
+                )
+
             duration_histogram_old.record(
                 max(round(duration_s * 1000), 0), duration_attrs_old
             )
         if duration_histogram_new:
             duration_attrs_new = otel_wsgi._parse_duration_attrs(
                 attributes, _HTTPStabilityMode.HTTP
             )
+
+            if wrapped_app_environ.get(_ENVIRON_REQUEST_ROUTE_KEY, None):
+                duration_attrs_new[SpanAttributes.HTTP_ROUTE] = (
```

Comment:
I think would be nice to use the new semconv symbol
from opentelemetry.semconv.attributes.http_attributes import HTTP_ROUTE
```suggestion
                duration_attrs_new[HTTP_ROUTE] = (
```

Reply:
Done!

Reply:
@GonzaloGuasch since we already did the semconv transition for flask, here I would keep 
`from opentelemetry.semconv.attributes.http_attributes import HTTP_ROUTE` symbol

Reply:
Done!

Reply:
with this changes do you see this PR ready to be merge? :)